plugins/wallabag.koplugin/main.lua

```diff
@@ -904,18 +1161,24 @@ function Wallabag:addArticle(article_url)
         ["Authorization"] = "Bearer " .. self.access_token,
     }
 
-    return self:callAPI("POST", "/api/entries.json", headers, body_JSON, "")
-end
+    return self:callAPI("POST /api/entries.json", headers, body_JSON) == true
```

Comment:
Could you clarify what exactly is going on in all of these? The signature remains `function Wallabag:callAPI(method, url, headers, body, filepath, quiet)` afaict.